soccerdata/fbref.py

```diff
@@ -1125,39 +1125,7 @@ def _concat(dfs: List[pd.DataFrame], key: List[str]) -> pd.DataFrame:
             columns.loc[mask, 1] = ""
             df.columns = pd.MultiIndex.from_tuples(columns.to_records(index=False).tolist())
 
-    # all dataframes should now have the same length and level 1 columns
     if len(all_columns) and all_columns[0].shape[1] == 2:
-        for i, columns in enumerate(all_columns):
-            if not columns[1].equals(all_columns[0][1]):
-                res = all_columns[0].merge(columns, indicator=True, how='outer')
-                raise RuntimeError(
-                    (
-                        "Cannot merge the data for {first} and {cur}.\n\n"
-                        + "The following columns are missing in {first}: {extra_cols}.\n\n"
-                        + "The following columns are missing in {cur}: {missing_cols}.\n\n"
-                        + "Please try to scrape the data again with caching disabled."
-                    ).format(
-                        first=dfs[0].iloc[:1][key].values,
-                        cur=dfs[i].iloc[:1][key].values,
-                        extra_cols=", ".join(
-                            map(
-                                str,
-                                res.loc[res['_merge'] == "left_only", [0, 1]]
-                                .to_records(index=False)
-                                .tolist(),
-                            )
-                        ),
-                        missing_cols=", ".join(
-                            map(
-                                str,
-                                res.loc[res['_merge'] == "right_only", [0, 1]]
-                                .to_records(index=False)
-                                .tolist(),
-                            )
-                        ),
-                    ),
-                )
-
```

Comment:
Please put this back.

Reply:
If we raise an error, then the code will never reach [this line](https://github.com/probberechts/soccerdata/pull/309/files#diff-96b59d76bf9a0ad4421d6247bff7992ddb7ff4fa5c44fa748854b16d875b756aR1147). If we are going to deal with the case where the columns are not the same later in the code, there's no reason to make sure they're the same here.